### PR TITLE
Don't check for code size in memorygrow tests when wasm in enabled

### PR DESCRIPTION
**Pull request by sbc100**
This check only makes sense for asm.js AFIACT

### CODE REVIEW

**Comment by sbc100** [comment]
@kripken PTAL

**Comment by sbc100** [comment]
@kripken can you take look?  This fixed some of the final remaining lld test failures.

File: tests/test_core.py [hunk 1]
```diff
@@ -1757,7 +1757,7 @@ def test_memorygrowth(self):
     self.do_run(src, '*pre: hello,4.955*\n*hello,4.955*\n*hello,4.955*')
     win = open('src.cpp.o.js').read()
 
-    if '-O2' in self.emcc_args:
+    if '-O2' in self.emcc_args and not Settings.WASM:
```
Comment (kripken) [inline tests/test_core.py]: can use `self.is_wasm()` for these

**Comment by kripken** [comment]
Ah, I think in asm2wasm it's just measuring a tiny amount of non-compiled code that is in that area, instead of comparing the actual code. So by luck it happens to pass. Good to avoid that with this PR.

File: tests/test_core.py (hunk 1 above)
Comment (sbc100) [reply tests/test_core.py]: Sounds reasonable.  I'm not sure what the difference is.  Perhaps we should change the occurrences of Settings.WASM in this file too (in a followup PR).

**Pull request merged**